config/daemonconfig/fuse.go

```diff
@@ -27,6 +27,7 @@ type FuseDaemonConfig struct {
 	EnableXattr     bool          `json:"enable_xattr,omitempty"`
 	AccessPattern   bool          `json:"access_pattern,omitempty"`
 	LatestReadFiles bool          `json:"latest_read_files,omitempty"`
+	Warmup          uint64        `json:"warmup"`
```

Comment:
Please also add a comment line here to tell it's experimental.

Reply:
Got it. Fixed.